spec/function.dd

```diff
@@ -1803,6 +1785,53 @@ void venus()
 }
 ---
 
+$(H4 $(LNAME2 struct-return-methods, Struct Return Methods))
+
+        $(P Struct non-static methods marked with the `return` attribute ensure the returned
+        reference will not outlive the struct instance.
+        )
+
+$(SPEC_RUNNABLE_EXAMPLE_FAIL
+---
+struct S
+{
+    private int x;
+    ref int get() return { return x; }
+}
+
+ref int escape()
+{
+    S s;
+    return s.get(); // Error: escaping reference to local variable s
+}
+---
+)
+        $(P The `return` attribute can also be used for wrapper structs to limit
+        the lifetime of the returned value:
+        )
+
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+@safe:
+
+struct S
+{
+    private int* p;
+    int* get() return => p;
+}
+
+void f()
+{
+    int* p;
+    {
+        S s;
+        //p = s.get(); // deprecated
```

Comment:
For consistency with other code examples here, don't comment it out and call it an error. Besides, DIP25 is already the default as of 2.103.0, and as I understand it it makes this an error, not a deprecation.

Reply:
> For consistency with other code examples here, don't comment it out

Now done. (I often prefer not to use RUNNABLE_EXAMPLE_FAIL as it doesn't check which particular lines errored).

> and call it an error. Besides, DIP25 is already the default as of 2.103.0

OK. BTW the example compiler is still 2.102.

> , and as I understand it it makes this an error, not a deprecation
> the spec doesn't seem to mention the need of the preview flag anymore with other dip1000 errors either

It was removed in https://github.com/dlang/dlang.org/pull/3518. -dip1000 is still needed sometimes (even with git master), see my comment there.

Reply:
I've now removed RUNNABLE until -dip1000 is the default, so the example runner works as documented.